client/my-sites/importer/importing-pane.jsx

```diff
@@ -116,7 +116,14 @@ class ImportingPane extends React.PureComponent {
 	getHeadingText = () => {
 		return this.props.translate(
 			'Importing may take a while if your site has a lot of media, but ' +
-				"you can safely navigate away from this page if you need to: we'll send you a notification when it's done."
+				'you can safely navigate away from this page if you need to or ' +
+				'go to our {{a}}handy checklist{{/a}} to explore your settings and customizing options. ' +
+				"We'll send you a notification when it's done.",
```

Comment:
I wonder if `We'll send you a notification when it's done` is really that valuable. We don't otherwise tell the customer that we'll send them a notification in other scenarios.
If we eventually have some other way of easily showing the progress & status of the import then it'd be even less valuable.
Given the new size of this message, should we chop out this closing sentence?

<img width="702" alt="screen shot 2018-11-26 at 17 06 33" src="https://user-images.githubusercontent.com/4335450/49029892-babe6700-f19d-11e8-9ebf-66f9523001a0.png">

Reply:
I kind of like the notification blurb -- it pre-answers the question, "how will I know when it's finished?"